scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -846,6 +846,21 @@ cdef class RigidTransformation:
         This implements the exponential map that converts 6-dimensional real
         vectors to SE(3).
 
+        An exponential coordinate vector consists of 6 elements: the first 3 encode
+        rotation (and form a rotation vector) and the last 3 encode translation
+        (and form a translation vector for pure translations). The exponential
+        mapping can be expressed as matrix exponential ``T = exp(tau)``. Where ``T`` is
+        a 4x4 matrix representing a ridig transformation and ``tau`` is a 4x4 matrix
+        formed from the elements of an exponential coordinate vector::
+
+            tau = [  0 -rz  ry vx]
+                  [ rz   0 -rx vy]
+                  [-ry  rx   0 vz]
+                  [  0   0   0  1]
+
+        With ``rx, ry, rz`` denoting the rotation part and ``vx, vy, vz`` denoting the
```

Comment:
We've used `tx`, etc elsewhere in the docs

Reply:
For translation, yes. This is not exactly the same as translation.

Reply:
I've kept it as `v`, not to mix with actual translation vector.